### PR TITLE
Flink Kubernetes Operator 1.9.0

### PR DESCRIPTION
**Pull request by mateczagany**
I will update the date once the vote thread is concluded.

### CODE REVIEW

**Comment by gyfora** [comment]
@mateczagany please only include the changes you made, and do not generate the new website files. That makes it much easier to review. 

We usually include the generated htmls as a separate second commit once the review is done

File: docs/content/posts/2024-06-30-release-kubernetes-operator-1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- usage of HikariPool for JDBC connections to reduce database pressure
+- memory leak fixes
+- improved error handling
+- support setting autoscaler options at standalone level
+- fix autoscaling when tasks are not running
+
+
+### Chinese Documentation
+
+There has been a lot of effort to translate to translate the Flink Kubernetes Operator documentation into Chinese in this release.
```
Comment (1996fanrui) [inline docs/content/posts/2024-06-30-release-kubernetes-operator-1.9.0.md]:
```suggestion
There has been a lot of efforts to translate the Flink Kubernetes Operator documentation into Chinese in this release. 
```

`to translate` is written twice.

File: docs/content/posts/2024-06-30-release-kubernetes-operator-1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+
+### Standalone Autoscaler Improvements
+
+There were numerous improvements for the Standalone Autoscaler component, such as:
+- usage of HikariPool for JDBC connections to reduce database pressure
+- memory leak fixes
+- improved error handling
+- support setting autoscaler options at standalone level
+- fix autoscaling when tasks are not running
```
Comment (1996fanrui) [inline docs/content/posts/2024-06-30-release-kubernetes-operator-1.9.0.md]:
```suggestion
- Memory tuning will scale JVM metaspace before heap to fix OOM due to too little metaspace memory


### Standalone Autoscaler Improvements

There were numerous improvements for the Standalone Autoscaler component, such as:
- Usage of HikariPool for JDBC connections to reduce database pressure
- Memory leak fixes
- Improved error handling
- Support setting autoscaler options at standalone level
- Fix autoscaling when tasks are not running
```

Using upper case for first word

**Review by 1996fanrui — APPROVED** [review]
Thanks @mateczagany for the update!

LGTM

**Comment by gyfora** [comment]
@mateczagany could you please update the release date to July 2? then we can merge this

**Review by gyfora — APPROVED** [review]
Looks good @mateczagany , please squash your commits and regenerate the HTMLs in a separate (2nd) commit

**Pull request merged**